easybuild/easyconfigs/e/Eigen/Eigen-3.3.7.eb

```diff
@@ -15,8 +15,13 @@ source_urls = [BITBUCKET_SOURCE]
 sources = ['%(version)s.tar.bz2']
 checksums = ['9f13cf90dedbe3e52a19f43000d71fdf72e986beb9a5436dddcd61ff9d77a3ce']
 
-# stick to latest CMake 3.9.x, since more recent CMake versions require a C++ compiler that supports C++11,
-# which may not be available yet in older OSs (e.g. CentOS 6.x)
-builddependencies = [('CMake', '3.9.6')]
+# Any CMake > 2.8.5 will work. Reusing one from GCCcore from around the same time, because
+# building CMake at core level is often difficult. 
+# Change the GCCcore version if you want to reuse a different CMake you already have installed.
+builddependencies = [
+    ('GCCcore', '7.3.0'),
+    ('binutils', '2.31.1'),
```

Comment:
@Micket Please add a comment for both `GCCcore` and `binutils` to clarify why they need to be listed explicitly.

Reply:
done